megamek/src/megamek/server/Server.java

```diff
@@ -33129,11 +31933,11 @@ public void run() {
                     ConnectionWatchdog w = new ConnectionWatchdog(this, id);
                     watchdogTimer.schedule(w, 1000, 500);
                 }
-            } catch (InterruptedIOException iioe) {
+            } catch (InterruptedIOException ignored) {
                 // ignore , just SOTimeout blowing..
-            } catch (IOException ex) {
+            } catch (IOException ignored) { }
 
-            }
+            // TODO : Windchild Can this be removed?
```

Comment:
There's a bunch of commented out code below, can that be removed?

Reply:
yes